deicode/optspace.py

```diff
@@ -180,7 +180,7 @@ def _fit(self):
                                niter=self.iteration, tol=self.tol)
         solution = U.dot(s_).dot(V.T)
         explained_variance_ratio_ = np.diag(s_) / np.diag(s_).sum()
-        self.eigenvalues = np.diag(s_)
+        self.eigenvalues = np.diag(s_)[::-1]
         self.explained_variance_ratio = list(explained_variance_ratio_)[::-1]
```

Comment:
that's the culprit -- reversed one line, but not the other 😆

Reply:
@mortonjt Yes, a sneaky little bug snuck right by me. Thanks for pointing out!

Reply:
This is probably ok for now -- but the longterm solution is to sort all of the eigenvalues, then `np.argsort` all of the columns in the samples and features.

This will require a larger unittest with say 5 dimensions.  Can push PR later.